### ISSUE TITLE
Does not show up in mobile device

### ISSUE THREAD

**Issue by samarthagarwal** [ksachdeva/angular-swiper#25]
My swiper is not showing up in my mobile phone (Android 4.3 and 5.0/5.1). It shows up okay in browser. I am using it inside my Ionic App.

Here is my code.

`<div class="item item-image">
                <ks-swiper-container initial-slide="0" autoplay="2000" loop="true" show-nav-buttons="true" pagination-clickable="true">
                    <ks-swiper-slide class="swiper-slide" ng-repeat="image in product.images">
                        <img ng-src="{{image.src}}">
                    </ks-swiper-slide>
                </ks-swiper-container>
</div>`

**Comment by olzii** [comment]
We have same issue, did you find solution ?

**Comment by samarthagarwal** [comment]
No, But it works fine if you do not use a $scope variable for ng-repeat. I instead used Ionic Slide Box.

**Comment by HugoHeneault** [comment]
Slidebox doesn't show multiple sliders at the same time. I have this issue too. Any ideas? :cry:

**Comment by scripter-co** [comment]
+1 there is an issue with using `$scoped` variables. A workaround I've found is to re-initialise the observers once the slider is ready. You can do the following:

```
<ks-swiper-container direction="vertical" swiper="swiper" on-ready="onReadySwiper(swiper)">
```

and in your controller

```
$scope.onReadySwiper = function(swiper) {
        swiper.initObservers();
});
```

I'll take a further look to see what the actual solution is.

**Comment by jimmyleo85** [comment]
@scripter-co  Thanks!!!

**Comment by thomaskanzig** [comment]
Guys, swiper.initObservers(); don't work in Internet Explorar browser :(
I also use with ng-repeat, but the images dont show.

Help me please!!!!

**Comment by etoah** [comment]
@scripter-co Thanks!!!

**Comment by justwezzz** [comment]
@scripter-co Thanks!!!

**Comment by pepamonsegur** [comment]
@scripter-co Thanks!!!!

**Comment by emblem-nyc** [comment]
@scripter-co Thank you so much!

**Comment by WillisYe** [comment]
add ng-if, My code follow:
`<ks-swiper-container ng-if="home.bannerContent[0] && home.bannerContent[0].content.length>0" autoplay="3000" loop="true" on-ready="onReadySwiper(swiper)" pagination-is-active="true" >
            <ks-swiper-slide class="swiper-slide" ng-repeat="item in home.bannerContent[0].content">
                <a ng-href="{{item.linkUrl}}">
                    <img ng-src="{{item.imgUrl}}" alt="" />
                </a>
            </ks-swiper-slide>
        </ks-swiper-container>`

**Comment by mrugesh008** [comment]
@scripter-co  I tried using ` $scope.onReadySwiper = function(swiper) {
                swiper.initObservers();
            };`   . But I get an error saying `Object doesn't support property or method 'initObservers'`
